person.rb

```diff
@@ -1,21 +1,25 @@
 require './nameable'
 
 class Person < Nameable
+  attr_accessor :age, :name
+  attr_reader :id, :rentals
```

Comment:
`rentals` should also have a setter. For that kindly use `attr_accessor` instead of `attr_reader`.

Reply:
Thanks for the kind feedback. Resolved